api/v1alpha1/volumereplication_types.go

```diff
@@ -44,6 +44,9 @@ const (
 
 	// ReplicationFailure means the last replication operation failed
 	ReplicationFailure ReplicationState = "Failed"
+
+	// Replicated means the last replication operation is succeeded
+	Replicated ReplicationState = "Replicated"
```

Comment:
what is the diff b/w replicating and replicated here?

Reply:
replicating means it is in progress and replicated means replication is succeeded

Reply:
Replicated does not sound correct.  Replication will never go to the finish state right? from our side we have enabled the replication in the background the replication just now started and it's replicating till we disable it.

Reply:
removed it.